source/fundamentals/connection/connection-options.txt

```diff
@@ -21,51 +21,69 @@ parameters of the connection URI to specify the behavior of the client.
      - Specifies the minimum number of connections that must exist at
        any moment in a single connection pool.
 
+       | **Default**: ``0``
+
    * - **maxPoolSize**
      - integer
      - Specifies the maximum number of connections that a connection
        pool may have at a given time.
 
+       | **Default**: ``100``
+
    * - **waitQueueTimeoutMS**
      - integer
      - Specifies the maximum amount of time, in milliseconds that a
        thread may wait for a connection to become available.
 
+       | **Default**: ``120000`` (2 minutes)
+
    * - **serverSelectionTimeoutMS**
      - integer
      - Specifies the maximum amount of time, in milliseconds, the driver
        will wait for server selection to succeed before throwing an
        exception.
 
+       | **Default**: ``30000`` (30 seconds)
+
    * - **localThresholdMS**
      - integer
      - When communicating with multiple instances of MongoDB in a replica
        set, the driver will only send requests to a server whose
        response time is less than or equal to the server with the fastest
        response time plus the local threshold, in milliseconds.
 
+       | **Default**: ``15`` (15 milliseconds)
```

Comment:
issue: this isn't consistent with the other defaults where the parenthetical content is the conversion for the next unit up, and I think it's apparent that 15 is 15.

```suggestion
       | **Default**: ``15``
```